pkg/nsx/services/common/policy_tree.go

```diff
@@ -206,6 +206,7 @@ var (
 	PolicyResourceStaticRoutes                  = PolicyResourceType{ModelKey: ResourceTypeStaticRoutes, PathKey: "static-routes"}
 	PolicyResourceVpcSubnet                     = PolicyResourceType{ModelKey: ResourceTypeSubnet, PathKey: "subnets"}
 	PolicyResourceVpcSubnetPort                 = PolicyResourceType{ModelKey: ResourceTypeSubnetPort, PathKey: "ports"}
+	PolicyResourceDhcpStaticBinding             = PolicyResourceType{ModelKey: ResourceTypeDhcpV4StaticBindingConfig, PathKey: "dhcp-static-binding-configs"}
```

Comment:
Could you check if those functions needed to be updated for DhcpStaticBinding? It would be better if the cleanup case can be test.

https://github.com/vmware-tanzu/nsx-operator/blob/fe03e64c56f6599911e34ab026248bd65c3c4781/pkg/nsx/services/common/policy_tree.go#L25
https://github.com/vmware-tanzu/nsx-operator/blob/fe03e64c56f6599911e34ab026248bd65c3c4781/pkg/nsx/services/common/policy_tree.go#L65
https://github.com/vmware-tanzu/nsx-operator/blob/fe03e64c56f6599911e34ab026248bd65c3c4781/pkg/nsx/services/common/policy_tree.go#L105